updated iOS binary protection checks (#1925)
OWASP/owasp-mstg: Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

    [SNIP]
    ```

    This check works for all cases, including pure Swift apps.

### Dynamic Analysis

> Dynamically these checks can be performed using Objection tool. One such example is below:

Reviewer: ```suggestion
hese checks can be performed dynamically using Objection. Here's one example:
```

please add the link to the Tools chapter for #objection
